src/main.py

```diff
@@ -61,14 +53,34 @@ def handle_urls(url: str, method: str) -> None:
             check_result = check_ok(url)
         case 'ping':
             hostname = obtain_hostname(url)
-            check_result = my_ping(hostname)
+            check_result = check_ping(hostname)
     
     if not check_result:
-        tg_res = run(make_request_to_telegram(url, method))
-        print(tg_res)
+        handle_retries(url,method,5)
     else:
         print(f'✅ Execution of {method} with address {url} succeeded')
 
+def handle_retries(url: str, method: str, tries: int) -> None:
+
+    tg_res = run(make_request_to_telegram(url, method))
+
+    for mess in tg_res:
+        print(f"mess: {mess}")
+   
+        if not mess['ok']:
+            print(f"Error {str(mess['error_code'])}: {mess['description']}")
+
+            if mess['error_code'] == 429:
+                #too many requests
+                sleep(30)
+                print("Error 429")
+
+            if tries > 0:
+                handle_retries(url,method,tries-1)
```

Comment:
- [ ] Some spaces should be added
- [ ] This would work flawlessly if there would be only a single chat id: in case only one message would fail, each user would receive the same message multiple times